index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Luk, A man</title>
+</head>
+<body>
+    <header>
+        <div class="header"> 
+            <div class="headertext">Welcome</div>
+            <div>
+            <label for="trigger">
+            <input type="checkbox" class="trigger" id="trigger"/>
+            <div class="hamburger">
+                <span></span>
+                <span></span>
+                <span></span>
+            </div>
+        <nav>
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </label>
+</div>
+        </div>
+    </header>
+    <section>
+        <div class="background">
+        <div class="container">
+            <div class="maintext">
+                <h1>Hey there. I’m <br>Lukman
+                <span class="h1-highlight"><br>I’m a software <br>developer</span></h1>
+            <div class="subtext">
+                <p> I can help you build a product,
+                     feature or website. Look through some of my work
+                      and experience! If you like what you see and 
+                      have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                    <div class="social">
+                        <a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg" alt="twitter"></a>
+                        <a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a>
+                        <a href="#"><img src="images\medium.svg" alt="medium"></a>
+                        <a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a>
+                        <a href="#"><img src="images\angellist.svg"></a>
+        
+                    </div>
```

Comment:
- [ ] Kindly make sure to wrap your social media icons into `ul` tag and wrap each one into `li` as well in order to correctly format your HTML semantically. :wink: